### ISSUE TITLE
Add classes based on the existing model weight

### ISSUE THREAD

**Issue by Sherlock-hh** [facebookresearch/detectron2#2366]
## ❓ How to add classes based on the existing model weight using detectron2
I'm using "COCO-InstanceSegmentation/mask_rcnn_X_101_32x8d_FPN_3x.yaml"， I want to add some classes to recognize. Can I load the pre-trained model and start the training with a new dataset? 

will it affect the previous predictions? Since orange and apple are predicted correctly  I don't want to affect the the classes which already can recognize. 

Is this the right way to do it? or Let me know if we have better options
Describe what you want to do, including:
1. I want to add ballloon to recognize and mask, I have many pictures include balloon.
2. I want to detetct the mask and 2dbox of balloon:

## ❓ What does an API do and how to use it?
the model weight I used: "COCO-InstanceSegmentation/mask_rcnn_X_101_32x8d_FPN_3x.yaml"
and I used the pytorch version of detectron2

**Comment by ppwwyyxx** [comment]
See #1024, #2124
